views/newsletter-page.js

```diff
@@ -0,0 +1,83 @@
+
+$(document).ready(function(){
+	createNewsletterTemplate ();
+	enableSubmitClick ();
+})
+
+function createNewsletterTemplate () {
+	let template = $('#newsletter-page-template').html();
+	let compiledTemplate = Handlebars.compile(template);
+	$('#application').html(compiledTemplate());
+}
+
+function enableSubmitClick () {
+    let forms = document.getElementsByClassName('needs-validation');
+    let validation = Array.prototype.filter.call(forms, function(form) {
+    	let button = document.getElementById('button');
+		button.onclick = function () {
+			form.addEventListener('submit', function(event) {
```

Comment:
Why is the "submit" handler assigned **after** the user clicked some button?

And what is that "button" for? (The name doesn't say much. ;))